### PR TITLE
remote call to logger that were uninitialized

### PR DESCRIPTION
**Pull request by apre**
replace it with a print() call.

### CODE REVIEW

File: src/borg/xattr.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def get_all(path, follow_symlinks=True):
         libc_name = 'libc.dylib'
     else:
         msg = "Can't find C library. No fallback known. Try installing ldconfig, gcc/cc or objdump."
-        logger.error(msg)
+        print(msg, file=sys.stderr)  # logger isn't initialized at this stage
```
Comment (ThomasWaldmann) [inline src/borg/xattr.py]: oh, oops.

so you found that in a traceback because it could find your C library and then it crashed when trying to log that?

Comment (apre) [inline src/borg/xattr.py]: Hi Thomas, 
This is exactly the case. It occured while I was hacking the codebase for own experiments.
Best regards,

Adrien

**Comment by ThomasWaldmann** [comment]
Hmm, does the problem also exist in 1.0-maint branch?

File: src/borg/xattr.py (hunk 1 above)
Comment (ThomasWaldmann) [inline src/borg/xattr.py]: Ehrm, what is your platform on which you triggered that problem?

Comment (ThomasWaldmann) [inline src/borg/xattr.py]: And is there some "usual" name for the libc on that platform? (see code above)

Comment (apre) [inline src/borg/xattr.py]: Here is my platform.
    Microsoft Windows [version 10.0.14393]
    (c) 2016 Microsoft Corporation. Tous droits réservés.
    D:\borg>python --version
    Python 3.5.2 :: Anaconda 2.5.0 (32-bit)

I am trying to port borg to a _native_ windows version, that's the reason why the 
 _libc_ is missing :)

Comment (enkore) [inline src/borg/xattr.py]: There has been some groundwork in the windows branch so far. With native-native you mean Python for Windows (+ MSVC, not MSYS) I guess?

Comment (apre) [inline src/borg/xattr.py]: oh, I have missed that branch fully. 
I will have a look into it.
I was trying to get it working with mingw32 compiler ( not so straightforward way, but it is for fun).

Also thinking about using _paramiko_ as replacement of external ssh pipe.

**Comment by ThomasWaldmann** [comment]
It does. So, please: collapse changesets into 1 and rebase onto 1.0-maint, PR against 1.0-maint.

**Comment by ThomasWaldmann** [comment]
@apre can you do that?

**Comment by apre** [comment]
Sure !
I will do that as soon as I find a free timesslice :) !

**Comment by ThomasWaldmann** [comment]
see #1840.

**Pull request closed without merging**